src/servers/main.cc

```diff
@@ -734,6 +759,15 @@ Parse(TRTSERVER_ServerOptions* server_options, int argc, char** argv)
 int
 main(int argc, char** argv)
 {
+#ifdef TRTIS_ENABLE_GPU
+  // Validate the format of specified minimum supported GPU compute capability
+  if (!ValidateMinComputeCapability()) {
```

Comment:
We want a compile-time failure. Can you see if you can use static_assert? Perhaps static_assert(TRTIS_MIN_COMPUTE_CAPABILITY >= 1.0) will be sufficient.